packages/cli/test/unit/ActiveWorkflowRunner.test.ts

```diff
@@ -51,7 +51,7 @@ const generateWorkflows = (count: number): WorkflowEntity[] => {
 	for (let i = 0; i < count; i++) {
 		const workflow = new WorkflowEntity();
 		Object.assign(workflow, {
-			id: i + 1,
+			id: (i + 1).toString(),
```

Comment:
thanks for this fix. [I've had this change in a branch for weeks](https://github.com/n8n-io/n8n/commit/84613648334ffeef1336d932ea98df5fba852f7f), but haven't had the opportunity to sneak it in a PR 🙏🏽

Reply:
in general, I'd really like us to move away from using `Object.assign` for creating entities, and use `repository.create` instead, as that's type-safe.